components/brave_wallet_ui/components/desktop/portfolio-account-item/index.tsx

```diff
@@ -25,7 +25,7 @@ export interface Props {
   action: () => void
   address: string
   fiatBalance: string
-  assetBalance: string
+  assetBalance: number
```

Comment:
I wonder if this is safe. It seems we are using floating-point numbers here because the values are not represented in base units (sats, Wei, etc). Or is it already the case?

Operations in Ethereum often require numbers that are outside the range of safe values to use in JavaScript. The preferred way is to use strings and convert to bignumber while doing conversions. Using base units also makes it easy to work with gas price, commonly represented in GWei. It's also the unit expected by contract methods.

Finally, we should be converting Wei to ETH only when the value is rendered in components. For everything else (computations, redux, etc), Wei should be the standard.

Reply:
yes please use strings in JS for all asset balances

Reply:
You are absolutely right! :) Currently the `ETH JSON Controller` is not setup up yet, so these value types will be updated as we get everything wired up!